web/src/LogPane.tsx

```diff
@@ -233,8 +233,10 @@ class LogPane extends Component<LogPaneProps, LogPaneState> {
       const key = "logLine" + i
 
       let shouldHighlight = false
+      let maybeHighlightRef = null
       if (highlight) {
         if (highlight.beginningLogID === i.toString()) {
+          maybeHighlightRef = this.highlightRef
```

Comment:
I'm surprised TypeScript let's you assign `React.RefObject<LogLineComponent>` to something that has type `null`?

Reply:
i think typescript is smart enough to type it correctly!
https://ts-ast-viewer.com/#code/DYUwLgBAHhC8EDsCuxgG4BQBLAZhAFALICGYAFgHQBOxCAJgPYC2+AlBAHwQAMFArOwDeGCKOhwIggL4YZGHEgQBjMFgYIIOfAE8AXIiRMARiCpCRYpeoDODUBWAMA5jtaYZWqG6A